operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/Context.java

```diff
@@ -2,13 +2,10 @@
 
 import io.fabric8.kubernetes.client.CustomResource;
 import io.javaoperatorsdk.operator.processing.event.EventList;
-import io.javaoperatorsdk.operator.processing.event.EventSourceManager;
 import java.util.Optional;
 
 public interface Context<T extends CustomResource> {
 
-  EventSourceManager getEventSourceManager();
```

Comment:
Why is this removed from the context?

Reply:
I think too that is not necessary there, just want to discuss the arguments.
So originally this looked a little bit different before the init method, the idea was that even dynamically would be possible to add event sources, or make some calls to event sources based on which resources is processed in which state. But users can set these instances directly to the class, so access those anyways possible. I basically agree with this.

Reply:
I'm trying to clean up / simplify things: this wasn't used and it introduced an unnecessary dependency. We can always re-add it later if it fills a need.